src/style.css

```diff
@@ -0,0 +1,142 @@
+p {
+  margin: 0;
+}
+
+h1 {
+  margin: 0;
+}
+
+h2 {
+  margin: 0;
+}
+
+body {
+  margin: 0 auto;
+}
+
+.block {
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+}
+
+.block__img {
+  height: 134px;
+  padding-top: 32px;
+  padding-right: 19px;
+  padding-left: 19px;
+}
+
+.block__title {
+  display: flex;
+  justify-content: start;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  padding-top: 40px;
+  padding-left: 16px;
+  padding-right: 16px;
+  color: #060b35;
+}
+
+.block__subtitle {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  padding-top: 4px;
+  padding-left: 16px;
+  padding-right: 16px;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 16px;
+  padding-left: 16px;
+  padding-right: 16px;
+}
+
+.stars__star {
+  background-image: url(../src/images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+  border-radius: 0.5px;
+
+}
+
+.stars__star--active :nth-child(-n + 4) {
+  background-image: url(../src/images/star-active.svg) ;
+}
```

Comment:
reuse all styles from Stars tasks,it should change correctly active stars if we change modifier